[2022-03-14 19:24:36,008] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-14 19:24:36,219] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-14 19:24:36,219] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 19:24:36,219] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-14 19:24:36,219] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 19:24:36,616] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_04> on 2022-03-13 00:00:00+00:00
[2022-03-14 19:24:36,673] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_04', 'scheduled__2022-03-13T00:00:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpqyj9p_0z', '--error-file', '/tmp/tmpyeqxik3p']
[2022-03-14 19:24:36,720] {standard_task_runner.py:77} INFO - Job 122: Subtask task_04
[2022-03-14 19:24:36,861] {standard_task_runner.py:52} INFO - Started process 5758 to run task
[2022-03-14 19:24:37,917] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_04 scheduled__2022-03-13T00:00:00+00:00 [running]> on host airflowvm
[2022-03-14 19:24:39,479] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_04
AIRFLOW_CTX_EXECUTION_DATE=2022-03-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-13T00:00:00+00:00
[2022-03-14 19:24:39,480] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-14 19:24:39,504] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-14 19:24:39,526] {subprocess.py:85} INFO - Output:
[2022-03-14 19:24:42,555] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-14 19:24:42,719] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_04, execution_date=20220313T000000, start_date=20220314T192436, end_date=20220314T192442
[2022-03-14 19:24:43,402] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 19:24:43,499] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
